webapp/channels/src/components/root/root.tsx

```diff
@@ -210,6 +213,8 @@ export default class Root extends React.PureComponent<Props, State> {
         this.a11yController = new A11yController();
 
         store.subscribe(() => applyLuxonDefaults(store.getState()));
+
+        this.performanceTelemetry = new PerformanceReporter(Client4, store);
```

Comment:
This should be aware of if you're logged in or not since the current spec for the API requires a session